src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -0,0 +1,129 @@
+import uk.gov.hmcts.reform.professionalapi.persistence.ProfessionalUserRepository;
+import uk.gov.hmcts.reform.professionalapi.util.ProfessionalReferenceDataClient;
+import uk.gov.hmcts.reform.professionalapi.util.Service2ServiceEnabledIntegrationTest;
+
+
+public class FindUsersByOrganisationTest extends Service2ServiceEnabledIntegrationTest {
+
+    @Autowired
+    private OrganisationRepository organisationRepository;
+
+    @Autowired
+    private ProfessionalUserRepository professionalUserRepository;
+
+    @Autowired
+    private ContactInformationRepository contactInformationRepository;
+
+    @Autowired
+    private DxAddressRepository dxAddressRepository;
+
+    @Autowired
+    private PaymentAccountRepository paymentAccountRepository;
+
+    private ProfessionalReferenceDataClient professionalReferenceDataClient;
+
+    @Before
+    public void setUp() {
+        professionalReferenceDataClient = new ProfessionalReferenceDataClient(port);
+        dxAddressRepository.deleteAll();
+        contactInformationRepository.deleteAll();
+        professionalUserRepository.deleteAll();
+        paymentAccountRepository.deleteAll();
+        organisationRepository.deleteAll();
+    }
+
+    @Test
+    public void can_retrieve_users_with_showDeleted_true_should_return_status_200() {
+        String organisationIdentifier = createOrganisationRequest();
+        updateOrganisation(organisationIdentifier, OrganisationStatus.ACTIVE);
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(organisationIdentifier,"True");
+        validateUsers(response);
+    }
+
+    @Test
+    public void can_retrieve_users_with_showDeleted_false_should_return_status_200() {
+        String organisationIdentifier = createOrganisationRequest();
+        updateOrganisation(organisationIdentifier, OrganisationStatus.ACTIVE);
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(organisationIdentifier,"False");
+        validateUsers(response);
+    }
+
+    @Test
+    public void can_retrieve_users_with_showDeleted_null_should_return_status_200() {
+        String organisationIdentifier = createOrganisationRequest();
+        updateOrganisation(organisationIdentifier, OrganisationStatus.ACTIVE);
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(organisationIdentifier,null);
+        validateUsers(response);
+
+    }
+
+    @Test
+    public void retrieve_users_with_pending_organisation_status_should_return_no_users_and_return_status_200() {
+        String organisationIdentifier = createOrganisationRequest();
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisation(organisationIdentifier,"True");
+        assertThat(response.get("http_status")).isEqualTo("200 OK");
+        assertThat(((List<ProfessionalUsersResponse>) response.get("users")).size()).isEqualTo(0);
+    }

```

Comment:
Remove memeber varaibles declaration and setup method which already declared in the super class.

Reply:
Moved code to Service2ServiceEnabledIntegrationTest to be reused by integration tests